publish_to_hugging_face.py

```diff
@@ -0,0 +1,41 @@
+from app_utils import hugging_face_utils
+
+import argparse
+import logging
+import sys
+
+if __name__ == "__main__":
+  parser = argparse.ArgumentParser(description="")
+  parser.add_argument("-e", "--experiment_dir", required=True, help="experiment dir", default=argparse.SUPPRESS)
```

Comment:
I would suggest to change this to `experiment_name` and then automatically build the correct path

Reply:
ROOT_DIR="/content/"

@dataclass
class Config(ConfigProblemBase):
    output_directory: str = "output/demo_fine_tuning/"
    experiment_name: str = "fine_tuning_experiment"
    llm_backbone: str = "EleutherAI/pythia-70m-deduped-v0"

!pipenv run python publish_to_hugging_face.py -e "EXPERIMENT_DIR" -d "cpu" -a "API_KEY" -r "USER_ID/MODEL_NAME"

@psinger Can you assist me in automatically creating the appropriate path based on the experiment's name?